corehq/apps/domain/templates/domain/manage_releases.html

```diff
@@ -44,9 +44,11 @@
                                     </div>
                                 </td>
                                 <td>
-                                    <a href="#" data-bind =
-                                    "click: toggleRestriction, text: actionText, visible: !ajaxInProgress()"></a>
-                                    <span class="spinner fa fa-spinner" data-bind="visible: ajaxInProgress"></span>
+                                    <button type="button" class="" data-bind="click: toggleRestriction">
```

Comment:
This is unstyled, it should use `btn` and `default-btn`.
<img width="204" alt="Screen Shot 2019-03-25 at 10 10 03 AM" src="https://user-images.githubusercontent.com/1486591/54930965-30390a80-4ee6-11e9-8c1b-ef918ea5636c.png">